src/js/pages/Search/index.js

```diff
@@ -20,8 +20,59 @@ export default class Search extends React.PureComponent {
     writer: PropTypes.string,
   };
 
+  constructor() {
+    super();
+    this.state = {
+      searchURL: '',
+    };
+    this.verseIdList = [];
+  }
+
+  setSearchUrl() {
+    const { q, type, offset, source } = this.props;
+    const isChatBot = type === SEARCH_TYPES.ASK_A_QUESTION;
+
+    if (isChatBot) {
+      const semanticApi = encodeURI(`${GURBANIBOT_URL}search/?query=${q}&count=100`);
+      try {
+        const semanticReq = fetch(semanticApi).then((response) => response.json());
+        semanticReq.then((semanticData) => {
+          this.verseIdList = semanticData.results.flatMap((dataObj) => {
+            const { VerseID, SourceID } = dataObj.Payload;
+            if (SourceID === source || source === 'all') {
+              return VerseID;
+            } else {
+              return [];
+            }
+          });
+          this.setState({ searchURL: `${API_URL}search-results/${this.verseIdList.toString()}?page=${offset}` });
+        });
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.error('err.message', err.message);
```

Comment:
Do we still need this error message in console at PROD?

Reply:
Yeah, we can setup another way to handle these errors in next releases. Currently the app uses multiple console.err statements.